.travis.yml

```diff
@@ -1,32 +1,26 @@
 language: node_js
-sudo: required
-before_script:
-  - npm install -g polymer-cli
-  - polymer install --variants
-  - >-
-    npm run update-types && git diff --exit-code || (echo -e
```

Comment:
why is this going away? we don't want it?

Reply:
(here in all the other PRs)

Reply:
(spoken offline) These were the typings and the formatting checks. Since we are going to use these travis files to check polymer 3 tests, then it would be useful to remove these as:

1. Polymer 3.0 typings do not exist yet and the gen js types does not work on them either
2. I don't know if the poylmer-modulizer output is clang-format compliant (I'm heavily leaning on "no")